<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWidget</class>
 <widget class="QWidget" name="ConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Rozpocznij zapis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelConnectButton">
       <property name="text">
        <string>Zakończ bez zapisywania</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>175</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="portListWidget"/>
     </item>
     <item>
      <widget class="QComboBox" name="channelWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="updatePortListButton">
       <property name="text">
        <string>Odśwież listę</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="connectionStateLabel">
          <property name="text">
           <string>Połączony: NIE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dataTransferLabel">
          <property name="text">
           <string>Transfer danych: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deviceTimeLabel">
          <property name="whatsThis">
           <string>Pokazuje czy na urządzeniu pomiarowym jest ustawiony prawidłowy czas</string>
          </property>
          <property name="text">
           <string>Czas urządzenia: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="startTimeLabel">
          <property name="text">
           <string>Czas rozpoczęcia: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="elapsedTime">
          <property name="text">
           <string>Czas pomiaru: --</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dziennik zdarzeń:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="logList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
